sbin/woeusb

```diff
@@ -1047,7 +1047,7 @@ create_uefi_ntfs_support_partition(){
         primary \
         fat16 \
         --  \
-        -1024s \
+        -2048s \
```

Comment:
I am looking to add support to calculate the numbers in run-time, but this would work for now.